### DISCUSSION TITLE
[Feature] Add SMTP send from name

### DISCUSSION THREAD

**Discussion by markrassamni** [docusealco/docuseal#45] [Ideas]
Currently you can only enter an address to send from. This could be improved by allowing users to also define the sender's name.
Then the recipient would get an email from `FirstName LastName <email@docuseal.co>` instead of just `email@docuseal.co`

**Comment by AlexBTurchyn** [comment]
@markrassamni In the last release (1.0.8), I started to include the company name. It is not what you wanted, but it is better than it was before.
